replace_numbers.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+
+
+def replace_numbers(num, operation, array, replace_num):
+    """
+    Get array and replace the values by replace_num if the give 'operation'(< or > or ==) comparison to num.
+    :param num:(int)
+    :param operation:(string)  Command: < or > or ==
+    :param array:(array) Given np array.
+    :param replace_num:(int)
+    :return:Array after replace values by the wanted operation.
+    """
+    print("Array before replace values by: " + operation)
+    print(array)
```

Comment:
No need to print inside function, seperat output from logic.
https://wiki.c2.com/?SeparateIoFromCalculation